sdk/resources/mgmt/src/main/java/com/azure/management/RestClientBuilder.java

```diff
@@ -29,7 +29,7 @@
  * A rest client builder.
  */
 @ServiceClientBuilder(serviceClients = RestClient.class)
-public final class RestClientBuilder {
+public final class RestClientBuilder implements Cloneable {
```

Comment:
Maybe we should change the name of `clone` method. Maybe `newBuilder`. (Because I didn't really want a `Cloneable` when adding that method.)

Reply:
done